About.html

```diff
@@ -0,0 +1,76 @@
+            <a href="#">Whatsap</a>
+            <a href="#">training</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="navlogo">
+            <a class="img-logo" href="index.html"><img src="img-folder/logo-image---00.png" alt=""></a>
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+                <li><a class="menu-item" href="#logo">logo</a></li>
+                <li><a class="menu-item" href="#vision">Vision</a></li>
+            </ul>    
+        </nav>
+    </header>    
+        <section class="grtittle">
+            <h1>Kamulete training center(Rwanda)</h1>
+            <hr class="linebar">
+            <div class="centerpar">
+                <p>kamulete trainig center for all people around the world,<br> we invite you to join our training center.
+                    kamulete training center was created in 2023 by kamulete bertin who is the CEO form Rwanda,<br> he found that to train people by knwowing many languages is very helps full and develops creativness in someone.
+                    with that this trainig center give you a diploma after complited the month acording to your program choosen,<br> that diploma is valid in each country around the word.
+                    we have more than 20 teachers by which we have collaboration and they helps us to give some online classes.
+                </p>
+            </div>
+          <div class="contactclass">
+            <p class="contactclass1">please contact us via email for more information.    </p>
+            <p class="contactclass2">email: Kamuletetrainingcenter988@gmail.com<br></p>
+            <p class="contactclass3">@250 kigali , palace (Rwanda)</p>
+          </div>
+        </section>
+        <section id="logo" class="classlog">
+            <h2>kamulete-training-center-logo</h2>
+            <hr class="linebar">
+            <p>the logo of kamulete traning center logo was decided from a competition from 12 january to 30january.
+                After finishing the competition voting and selection committe's judjjing process,<br> the logo from 20 years oldswon the prize-winning-logo for kamulete training center as follow
+            </p>
+            <img src="img-folder/logo-image.png" alt="">
+        </section>
+        <section id="vision" class="vissonsee">
+            <h2>see our vision and goals </h2>
+            <p>see how we working with our students</p>
+           <div>
+            <img class="Viision" src="img-folder/students1.jfif" alt="">
+            <img class="Viision" src="img-folder/students2.jfif" alt="">
+           </div>
+        </section>
```

Comment:
- [ ] This part of your about section is still missing some styles to look similar to the given design :+1: 
- [ ] First off, the heading of the section is not properly styled and has a different font color than what was used in the design. Same as the paragraph text. Please style these elements to look similar to the original design :+1: 
- [ ] Your images also look different that what was used in the original design. Please ensure that they have the same overlay as in the original design and have some text written on them similar to the original design  :100: 
- [ ] Lastly, kindly ensure that the elements have similar spacing, borders, shadows, and paddings to the original design.
- Your Design
![image](https://user-images.githubusercontent.com/102699958/225421652-e4142ea6-a4a6-4c32-884a-485e41f5874c.png)
- The Given Design
![image](https://user-images.githubusercontent.com/102699958/225421941-c5849479-7387-4e39-a468-99c21ef5a769.png)